superdesk/publish/formatters/__init__.py

```diff
@@ -112,6 +112,23 @@ def map_html_to_xml(self, element, html):
         if len(list(element)) == 0:
             SubElement(element, 'p').text = soup.get_text()
 
+    def pass_through(self, article):
```

Comment:
just wondering why it's on formatter, is it supposed to be used by specific formatter implementation?